coordinator/src/bin/coordinator.rs

```diff
@@ -34,6 +34,7 @@ const PROCESS_PROMETHEUS_METRICS: Duration = Duration::from_secs(10);
 const PROCESS_INCOMING_DLC_MESSAGES_INTERVAL: Duration = Duration::from_secs(5);
 const EXPIRED_POSITION_SYNC_INTERVAL: Duration = Duration::from_secs(300);
 const CLOSED_POSITION_SYNC_INTERVAL: Duration = Duration::from_secs(30);
+const UNREALIZED_PNL_SYNC_INTERVAL: Duration = Duration::from_secs(60);
```

Comment:
🤔 Every minute is quite often isn't it? How about every 10 minutes?

Reply:
I guess it depends how accurate we want a report that takes the unrealized pnl into account should be.
I'm happy to put 10 mins :)

Reply:
was changed to `600` (i.e. 10 mins)